virtcontainers/mount_test.go

```diff
@@ -218,7 +218,7 @@ func TestGetDeviceForPathBindMount(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	defer syscall.Unmount(dest, 0)
+	defer syscall.Unmount(dest, syscall.MNT_DETACH)
```

Comment:
Do you want to handle or log the error if the syscall to unmount fails

Reply:
I noticed some mountpoints left as garbage when local tests finished, syscall.MNT_DETACH should be safe to remove the garbage